playbooks/generic-clean_airship.yml

```diff
@@ -14,11 +14,11 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 #
-- hosts: airship-openstack-compute-workers
+- hosts: airship-openstack-compute-workers:soc-deployer
   gather_facts: no
   any_errors_fatal: true
   roles:
-    - { role: openstack-remove-compute, compute_node_name: "{{ inventory_hostname }}" }
+    - { role: openstack-remove-compute, compute_node_name: "{{ groups['airship-openstack-compute-workers'] }}" }
```

Comment:
How this will work ? 

groups is an array and it seems like compute_node_name is expected to be single host value

`               "airship-openstack-compute-workers": [
                    "linux-1pm5",
                    "linux-9nya",
                    "linux-xj84",
                    "linux-vrxc"
                ],
`